config/application.rb

```diff
@@ -28,6 +28,17 @@ class Application < Rails::Application
       # Application configuration should go into files in config/initializers
       # -- all .rb files in that directory are automatically loaded.
 
+      # Setup redis as the cachestore if settings for it are available.
+      # This is required for sidekiq, which uses redis to queue jobs.
+      config.cache_store = begin
```

Comment:
Given that we are enforcing use of Sidekiq, we should enforce the presence of the Redis config file, and have app boot fail if it is not present.